games/game_nomanssky.py

```diff
@@ -0,0 +1,26 @@
+# -*- encoding: utf-8 -*-
+
+from ..basic_game import BasicGame
+
+
+class NoMansSkyGame(BasicGame):
+
+    Name = "Mo Man's Sky Support Plugin"
+    Author = "Luca/EzioTheDeadPoet"
+    Version = "1.0.0"
+
+    GameName = "Mo Man's Sky"
+    GameShortName = "nomanssky"
+    GaneNexusHame = "nomanssky"
```

Comment:
GaneNexusHame?

Reply:
oh a typo, but iirc since it uses `GameShortName` if `GameNexusName` is not found and I used the same strings it should still work. (I must have missed that while coping a template I made myself)